src/site/Site.js

```diff
@@ -11,3 +11,4 @@ import "./SiteNavigation";
 import "./SeForm";
 import "./SiteSearch";
 import "./SocialMedia";
+import "../utilities/googlesearch";
```

Comment:
This is what was missing to actually add the google javascript to the main minified file